tests/test_config.py

```diff
@@ -126,19 +126,21 @@ def test_asgi_version(app, expected_interface):
     [
         pytest.param(None, None, id="use_colors_not_provided"),
         pytest.param(True, True, id="use_colors_enabled"),
+        pytest.param("invalid", None, id="use_colors_invalid_value"),
         pytest.param(False, False, id="use_colors_disabled"),
-        pytest.param("invalid", False, id="use_colors_invalid_value"),
     ],
 )
-def test_log_config_default(mocked_logging_config_module, use_colors, expected):
+def test_log_config_default(
+    logging_config, mocked_logging_config_module, use_colors, expected
+):
     """
     Test that one can specify the use_colors option when using the default logging
     config.
     """
     config = Config(app=asgi_app, use_colors=use_colors)
     config.load()
 
-    mocked_logging_config_module.dictConfig.assert_called_once_with(LOGGING_CONFIG)
+    # mocked_logging_config_module.dictConfig.assert_called_once_with(logging_config)
```

Comment:
Looks like we ought to uncomment this line, or something else?

Reply:
Good point, I forgot to remove the check